.github/workflows/pkgci_test_tensorflow_cpu.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI Test TensorFlow (CPU)
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+
+jobs:
+  linux_x86_64:
+    name: Linux (x86_64)
+    runs-on:
+      - self-hosted # must come first
+      - runner-group=${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
+      - environment=prod
```

Comment:
It is rarely used but this will remove the capability to test jobs on testing runners (e.g. runners with testing GCP templates). Just FYI

Reply:
Well, now that you mention it... I could switch this to use a GitHub-hosted runner. I was about ready to try https://github.com/openxla/iree/pull/16200 again now that these jobs are only using install folders / packages instead of the build directory.

Reply:
Okay, pushed a commit trying GitHub-hosted runners. I should be careful about crossing the streams like this...

The AMDGPU runners like `runs-on: nodai-amdgpu-w7900-x86-64` use this instead of the GCP runner stuff that exists in-tree in IREE:
* https://github.com/nod-ai/base-docker-images/blob/main/dockerfiles/amdgpu_ubuntu_jammy_ghr_x86_64.Dockerfile
* https://github.com/myoung34/docker-github-actions-runner

I haven't parsed through all the details myself, so I'm not sure what is involved in keeping those runners up to date on runner versions. I at least think those won't have an environment distinction like 'prod' for rollouts.

Reply:
Cool, standard GitHub-hosted runners worked. A bit slower but not by much (1m30s -> 2m).